packages/shared/src/domain_model/Candidate.ts

```diff
@@ -10,4 +10,5 @@ export interface Candidate {
     candidate_url?: string; // link to info about candidate
     partyUrl?:      string; // link to info about party
     photo_filename?:string; // link to info about party
+    score?:         number; // score for the candidate
```

Comment:
Rather than add score here, maybe we should create a new type that extends from the candidate that is only used in the vote pages.